views/captcha.html

```diff
@@ -19,20 +19,20 @@
         <img id=image src="/captchas/{{.CaptchaID}}.png" alt="your captcha image">
     </div>
 
-    <form action="/verifyhuman" method="post" class="w-100 form form--narrow-inputs" autocomplete="off">
+    <form action="/verifyhuman" id="captcha-form" method="post" class="w-100 form form--narrow-inputs" autocomplete="off">
         <div class="col-12 mb-4 text-left center-block">
             <div class="form-group row mb-0 align-items-center">
                 <label class="pr-4" for="captchaSolution">Text&nbsp;in&nbsp;picture:</label>
                 <div>
-                    <input name=captchaSolution class="form-control">
+                    <input name=captchaSolution required class="form-control">
```

Comment:
Can we change this to 
```suggestion
                    <input name=captchaSolution required type="text" maxlength="6" pattern="\d{6}" class="form-control">
```

Reply:
Yeah we can look at this, could you create a new issue please?